properties/color/base.json

```
@@ -22,19 +22,28 @@
       "value": "#ffffff"
     },
     "orange": {
-      "value": "#ff7f3f"
+      "value": "#ff8912"
```

Lo mismo para el orange, el bueno es este:
```suggestion
      "value": "#fa915a"
```